src/eko/couplings.py

```diff
@@ -272,20 +269,19 @@ def expanded_qed(ref, order, nf, lmu):
         QED coupling at target scale :math:`a_em(\mu_R^2)`
     """
     res_aem = ref
-    beta_vec = [beta_qed((0, 2), nf)]
-    for i in range(2, order + 1):
-        beta_vec.append(b_qed((0, i + 1), nf))
     # LO
     if order == 1:
-        res_aem = exact_lo(ref, beta_vec[0], lmu)
+        res_aem = exact_lo(ref, beta0, lmu)
     # NLO
     if order == 2:
-        res_aem = expanded_nlo(ref, beta_vec[0], beta_vec[1], lmu)
+        res_aem = expanded_nlo(ref, beta0, b_vec[1], lmu)
     return res_aem
 
 
 @nb.njit(cache=True)
-def couplings_expanded_alphaem_running(order, couplings_ref, nf, scale_from, scale_to):
+def couplings_expanded_alphaem_running(
+    order, couplings_ref, nf, scale_from, scale_to, decoupled_running
+):
     r"""Compute coupled expanded expression of the couplings for running alphaem.
```

Comment:
If this is already doing the decoupled case, please update the doc here and below

Reply:
Yes I added it but if you notice for the moment it is always set to `False`.